transport/src/main/java/io/scalecube/transport/Transport.java

```diff
@@ -177,19 +181,20 @@ protected void initChannel(Channel channel) {
       }
     });
 
-    ChannelFuture bindFuture = server.bind(localEndpoint.socketAddress());
+    ChannelFuture bindFuture = server.bind(localEndpoint.port());
```

Comment:
We don't want to limit on which interface you bind. At least by default you should bind to any interface (0.0.0.0) if it wasn't configured otherwise.